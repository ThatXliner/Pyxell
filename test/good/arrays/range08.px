
print ['c'..'d', "xyz"]
print [-1, 0..4^-2 by 1/32]
print [5..3.5 by -0.75, 0...2, 1/2]
