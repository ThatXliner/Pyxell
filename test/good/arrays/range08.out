[c, d, xyz]
[-1, 0, 1/32, 1/16]
[5.0, 4.25, 3.5, 0.0, 1.0, 0.5]
